Makefile

```diff
@@ -140,9 +140,6 @@ build-release:
 	mkdir -p bin/docker-archive/local-volume-provider
 	skopeo copy docker://replicated/local-volume-provider:${LVP_VERSION} docker-archive:bin/docker-archive/local-volume-provider/${LVP_VERSION}
 
-	mkdir -p bin/docker-archive/local-volume-fileserver
```

Comment:
why is this removed? i think this is needed for velero to function (getting backup logs, etc...)